recipes/google-cloud-cpp-core/meta.yaml

```diff
@@ -0,0 +1,364 @@
+{% set version = "2.17.0" %}
+
+{%- macro test_subpackage(feature) %}
+commands:
+  - test -f $PREFIX/lib/libgoogle_cloud_cpp_{{ feature }}.{{ version }}.dylib  # [osx]
+  - test -f $PREFIX/lib/libgoogle_cloud_cpp_{{ feature }}.so.{{ version }}  # [linux]
+  - test ! -f $PREFIX/lib/pkgconfig/google_cloud_cpp_{{ feature }}.pc  # [not win]
+  - test ! -f $PREFIX/include/google/cloud/version.h  # [not win]
+  - test ! -f $PREFIX/lib/cmake/google_cloud_cpp_{{ feature }}/google_cloud_cpp_{{ feature }}-config.cmake  # [not win]
+  - if exist %LIBRARY_PREFIX%\\include\\google\\cloud\\version.h exit 1  # [win]
+  - if exist %LIBRARY_PREFIX%\\lib\\cmake\\google_cloud_cpp_{{ feature }}\\google_cloud_cpp_{{ feature }}-config.cmake exit 1  # [win]
+{%- endmacro %}
+
+{%- macro test_devel_subpackage(feature) %}
+commands:
+  - ./run_test_feature.sh   # [not win]
+  - ./run_test_feature.bat  # [win]
+requires:
+  - {{ compiler('cxx') }}
+  - cmake
+  - ninja
+files:
+  - run_test_feature.sh
+  - run_test_feature.bat
+source_files:
+  - google/cloud/{{ feature }}/quickstart/*.cc
+  - google/cloud/{{ feature }}/quickstart/CMakeLists.txt
+{%- endmacro %}
+
+{%- macro standard_subpackage(feature) %}
+script: build-libgoogle-cloud.sh   # [not win]
+script: build-libgoogle-cloud.bat  # [win]
+build:
+  run_exports:
+    # patch versions guaranteed to be ABI-compatible, see
+    # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/adr/2023-05-03-patch-releases-are-ABI-compatible.md
+    - {{ pin_subpackage("libgoogle-cloud-common", max_pin="x.x") }}
```

Comment:
Not sure if we need the general run-export on the non-`devel` feature packages. Basically there needs to be at least one devel package in host anyway to build correctly, and those have the right run-export already.

Reply:
Done I think.